odh/base/monitoring/overrides/prometheus-operator/overlays/alerts/prometheus-rules.yaml

```diff
@@ -11,14 +11,32 @@ spec:
     - name: Operate First Availability
       rules:
         - alert: Prometheus Service Down
-          expr: up{service="prometheus-operated"} == 0
+          expr: count(up{service="prometheus-operated", pod=~"prometheus-odh-monitoring.*"} == 0) > 1
           for: 5m
           annotations:
             summary: "Prometheus service is no longer available"
             severity: "HIGH"
         - alert: Prometheus Alertmanager Down
-          expr: up{service="alert-manager"} == 0
+          expr: absent(up{service="alert-manager"} == 1)
           for: 5m
           annotations:
             summary: "Prometheus Alertmanager is no longer available"
             severity: "HIGH"
+        - alert: Grafana Service Down
+          expr: absent(up{service="grafana-service"} == 1)
+          for: 5m
+          annotations:
+            summary: "Grafana Service is no longer available"
+            severity: "HIGH"
+        - alert: Grafana Operator
+          expr: absent(up{service="grafana-operator-metrics"} == 1)
```

Comment:
I feel like the grafana operator being unavailable is less bad than grafana/prometheus/argocd/alertmanager not being available.

Reply:
so just have this in the grafana dashboard, but not be alerted on it?

Reply:
Maybe just lowering the severity to `LOW` would be fine?

Reply:
updated